### ISSUE TITLE
TiKV fully in-memory

### ISSUE THREAD

**Issue by raphaelauv** [tikv/tikv#17336]
## Question

 I would like to use TiKV fully in-memory without persistence is that possible ? ( I see that I can't disable the rocks db WAL )
 
 Thanks

**Comment by LykxSassinator** [comment]
Nope, TiKV do not have official configurations to close the persistence.

FYI, you may want to consider using `tmpfs` or `ramfs` to mount a block of RAM as a dedicated disk for your specific use case if possible.
